### ISSUE TITLE
Wording for different CAPACITY variables

### ISSUE THREAD

**Issue by sandrinecharousset** [openENTRANCE/nomenclature#21]
The word Capacity may refer to different concepts.
It already exist in IAMC where it stands for Installed capacity  (eg Capacity|Electricity: description: Total installed electricity generation capacity,  unit: GW) ; It is also used for defining the installed capacity per technology (eg Capacity|Electricity|Wind|Offshore)

@erikfilias commented in #12 (closed pull request)
I have a doubt about declaring the maximum capacity of a pumped-storage hydropower plant. It could be as:
Maximum Capacity|Secondary Energy|Electricity|Hydro|Pumped Storage|Power Plant 1
Where the Maximum Capacity is a variable that belongs to Power Plant 1 and this plant is a pumped-storage hydropower plant.

where Maximum Capacity would refer to the maximum possible power generation of a given plant, which is a very different concept.

@danielhuppmann gave following answer:
Yes, in principle that is how this could be added - but a few considerations and comparison to the IAMC 1.5°C variable list (see here), where a similar variable is called Capacity|Electricity|Hydro
1-Having both Secondary Energy and Electricity is redundant for this type of information (there is no electricity of final energy of a hydro plant), so it can be omitted
2-Maximum Capacity is not clear in the context of a pumped-hydro storage plant - is it storage capacity (reservoir), generation capacity, or pumping capacity?
3-For other power plants, "maximum" generation capacity is redundant in the usual terms (that's why we only used Capacity in the IAMC 1.5°C context).
So in short, I would suggest to use this more concise version:
Capacity|Electricity|Hydro|Pumped Storage|Power Plant 1 for maximum generation capacity
Reservoir Capacity|Electricity|Hydro|Pumped Storage|Power Plant 1 for reservoir capacity

I personnally think that Capacity should be used only when describing the installed capacity of a certain kind of technology. Maximum capacity may be used in some investment models to give the max constraint (the 'potential'). Moreover, the maximum active power of an electricity generation asset is (in my opinion) a 'technology ' variable (a technical constraint). I would be in favour of using different words for different concepts. 
In plan4res we use Maximum Power or Maximum Active Power which is more accurate.
